controllers/keystoneapi_controller.go

```diff
@@ -317,10 +323,8 @@ func (r *KeystoneAPIReconciler) reconcileInit(
 	}
 	if dbSyncjob.HasChanged() {
 		instance.Status.Hash[keystonev1.DbSyncHash] = dbSyncjob.GetHash()
-		if err := r.Client.Status().Update(ctx, instance); err != nil {
-			return ctrl.Result{}, err
-		}
 		r.Log.Info(fmt.Sprintf("Job %s hash added - %s", jobDef.Name, instance.Status.Hash[keystonev1.DbSyncHash]))
+		return ctrl.Result{}, nil
```

Comment:
With this we'll only reconcile on the next event, right? for running jobs we can expect one to come. just wondering if there can be a situation in one of the bellow `return ctrl.Result{}, nil` that this might not happen? It might make sense to set `Requeue` to true ? [1]

[1] https://pkg.go.dev/sigs.k8s.io/controller-runtime@v0.13.1/pkg/reconcile#Result